ci/Jenkinsfile.linux

```diff
@@ -73,6 +73,7 @@ pipeline {
     STATUS_CLIENT_TARBALL = "pkg/${utils.pkgFilename(ext: 'tar.gz', arch: getArch())}"
     /* prevent sharing cache dir across different jobs */
     GO_GENERATE_FAST_DIR = "${env.WORKSPACE_TMP}/go-generate-fast"
+    SENTRY_PRODUCTION = "${utils.isReleaseBuild() ? 'true' : 'false'}"
```

Comment:
How is this env variable actually checked? Are we sure a `false` string won't just be treated as true?

Reply:
It's checked for `true` and `1`:
https://github.com/status-im/status-go/blob/987a9e8707e97c1757f87f621c036d671bb046fe/internal/sentry/params.go#L34